### PR TITLE
feat: update images in kustomzie thanos sepolia

### PR DESCRIPTION
**Pull request by 0x6e616d**

### CODE REVIEW

**Review by rlgns98kr — APPROVED** [review]
Thank you very much! 😍

File: .github/workflows/release-docker-nightly.yml [hunk 1]
```diff
@@ -360,29 +360,29 @@ jobs:
   #       with:
   #         github-token: ${{ secrets.ACCESS_TOKEN }}
 
-  #     - name: Update titan-op-node Kubernetes resources
```
Comment (boohyung) [inline .github/workflows/release-docker-nightly.yml:363]: please uncomment if we need

Comment (0x6e616d) [reply .github/workflows/release-docker-nightly.yml:363]: yes, I follow @rlgns98kr to uncomment these lines. We will uncomment if needed

Steven's PR: https://github.com/tokamak-network/tokamak-thanos/pull/125

Comment (boohyung) [reply .github/workflows/release-docker-nightly.yml:363]: Ok. We will make k8s resources of thanos-sepolia-nightly and update this CI.

**Review by boohyung — APPROVED** [review]

**Pull request merged**